data/data/aws/vpc/vpc-public.tf

```diff
@@ -43,13 +43,28 @@ resource "aws_route" "igw_route" {
   }
 }
 
+resource "aws_route" "igw_route_v6" {
+  count = var.use_ipv6 == true && var.vpc == null ? 1 : 0
+
+  destination_ipv6_cidr_block = "::/0"
+  route_table_id              = aws_route_table.default[0].id
+  gateway_id                  = aws_internet_gateway.igw[0].id
+
+  timeouts {
+    create = "20m"
+  }
+}
+
 resource "aws_subnet" "public_subnet" {
   count = var.public_subnets == null ? length(var.availability_zones) : 0
 
   vpc_id            = data.aws_vpc.cluster_vpc.id
   cidr_block        = cidrsubnet(local.new_public_cidr_range, 3, count.index)
   availability_zone = var.availability_zones[count.index]
 
+  ipv6_cidr_block                 = var.use_ipv6 == true ? cidrsubnet(data.aws_vpc.cluster_vpc.ipv6_cidr_block, 8, count.index + length(var.availability_zones)) : ""
```

Comment:
https://github.com/openshift/installer/blob/master/data/data/aws/vpc/vpc-public.tf#L50 is how the IPv4 address is calculated.

my preference would be to keep this calculated similar looking if possible.

Reply:
I've added some comments to hopefully help explain why this is the way it is.  Let me know if you have a more specific suggestion.